### ISSUE TITLE
Screen Reader Not Reading Label of Radio Component

### ISSUE THREAD

**Issue by MozesKadar** [gluestack/gluestack-ui#2544]
### Description

I'm experiencing an issue with the Radio component where the screen reader reads the value field instead of the label. When I set the accessibilityLabel property, the screen reader still doesn't read it correctly.

### CodeSandbox/Snack link

-

### Steps to reproduce

1. Add a Radio component with a label and a value.
2. Enable a screen reader and navigate to the Radio component.
3. Observe that the screen reader reads the value instead of the label or accessibilityLabel.

### gluestack-ui Version

1.1.48

### Platform

- [ ] Expo
- [X] React Native CLI
- [ ] Next
- [ ] Web
- [ ] Android
- [ ] iOS

### Additional Information

Expected Behavior: The screen reader should read the label or, if provided, the accessibilityLabel for the Radio component.

Actual Behavior: The screen reader reads the value field regardless of the presence of label or accessibilityLabel.

**Comment by Viraj-10** [comment]
Hey @MozesKadar, Thanks for reporting the issue we will have a look.

**Comment by Sanchitv3** [comment]
We’ve just released a fix for this issue in **gluestack-ui v5 alpha**. The screen reader should now correctly pick up the `label` / `accessibilityLabel` for the Radio component.

Please try it out and let us know if you still face any issues.
